src/lib.rs

```diff
@@ -317,7 +323,7 @@ fn test_resize_policy() {
 /// }
 /// ```
 #[derive(Clone)]
-pub struct HashMap<K, V, S = RandomState> {
+pub struct HashMap<K, V, S = AdaptiveState> {
```

Comment:
Yes, this change can't be added to the std library. Is this library a drop-in replacement for std's HashMap?